pipelines/pingcap/tidb/latest/merged_integration_typeorm_test.groovy

```diff
@@ -0,0 +1,159 @@
+                    }
+                } 
+                stages {
+                    stage("Test") {
+                        options { timeout(time: 40, unit: 'MINUTES') }
+                        steps {
+                            dir('tidb') {
+                                cache(path: "./bin", filter: '**/*', key: "binary/pingcap/tidb/merged_integration_typeorm_test/rev-${BUILD_TAG}") {
+                                    sh label: 'tidb-server', script: 'ls bin/tidb-server && chmod +x bin/tidb-server && ./bin/tidb-server -V'  
+                                    sh label: 'tikv-server', script: 'ls bin/tikv-server && chmod +x bin/tikv-server && ./bin/tikv-server -V'
+                                    sh label: 'pd-server', script: 'ls bin/pd-server && chmod +x bin/pd-server && ./bin/pd-server -V'  
+                                }
+                            }
+                            dir('tidb-test') {
+                                cache(path: "./", filter: '**/*', key: "ws/${BUILD_TAG}/tidb-test") {
+                                    sh """
+                                        mkdir -p bin
+                                        cp ${WORKSPACE}/tidb/bin/* bin/ && chmod +x bin/*
+                                        ls -alh bin/
+                                    """
+                                    container("nodejs") {
+                                        sh label: "test_params=${TEST_PARAMS} ", script: """
+                                            #!/bin/bash
+                                            set -- \${TEST_PARAMS}
+                                            TEST_DIR=\$1
+                                            TEST_SCRIPT=\$2
+                                            echo "TEST_DIR=\${TEST_DIR}"
+                                            echo "TEST_SCRIPT=\${TEST_SCRIPT}"
+
+                                            if [[ "${TEST_STORE}" == "tikv" ]]; then
+                                                echo '[storage]\nreserve-space = "0MB"'> tikv_config.toml
+                                                bash ${WORKSPACE}/scripts/PingCAP-QE/tidb-test/start_tikv.sh
+                                                export TIDB_SERVER_PATH="${WORKSPACE}/tidb-test/bin/tidb-server"
+                                                export TIKV_PATH="127.0.0.1:2379"
+                                                export TIDB_TEST_STORE_NAME="tikv"
+                                                cd \${TEST_DIR} && chmod +x *.sh && \${TEST_SCRIPT}
+                                            else
+                                                export TIDB_SERVER_PATH="${WORKSPACE}/tidb-test/bin/tidb-server"
+                                                export TIDB_TEST_STORE_NAME="unistore"
+                                                cd \${TEST_DIR} && chmod +x *.sh && \${TEST_SCRIPT}
+                                            fi
```

Comment:
It's better push the script in `tidb-test` repo.

Reply:
I'm not sure why previous tests were written like this